docs/configuration/recording_rules.md

```diff
@@ -132,6 +132,9 @@ annotations:
   [ <labelname>: <tmpl_string> ]
 ```
 
+See also the
+[best practices for naming recording rules](https://prometheus.io/docs/practices/rules/#recording-rules).
```

Comment:
Nit: It's not the name of the rule, but the name of the metric, so maybe "...for naming metrics created by recording rules"?

Reply:
Ohhh yeah you're right, this also avoids confusion with naming metrics in general. Thanks for the catch! Changed in this [commit](https://github.com/prometheus/prometheus/pull/11696/commits/fc039c52dc11234acfa903d5d7ba74bef24a71cd)